server/README.md

```diff
@@ -149,19 +153,24 @@ Then run the following commands to install Hatohol:
 
 ## How to start
 (0.1) Run MySQL server for storing configuration.
+Type the following comannds to run MySQL server.
+
+    # mysqld
```

Comment:
Directly `mysqld` execution is not welcome.

Ordinary Linux users execute `mysqld` as daemon mode with `Upstart` or `Systemd` and so on.

Thus, this command is inadequate.